function handleError(context, error) {
    console.error(`Error in ${context}:`, error);

}

module.exports = { handleError };
